Breadcrumbs.py

```diff
@@ -1,12 +1,18 @@
 # -*- coding: utf-8 -*-
-import sublime, sublime_plugin
 import math
+import re
+import html
+
+import sublime
+import sublime_plugin
 
 try:
   xrange
 except NameError:
   xrange = range
 
+settings = sublime.load_settings('Breadcrumbs.sublime-settings')
```

Comment:
The View class has a `scope_name(point)` method that returns the scope at the current position. Inline javascript in a html file would return something useful including `source.js`:

<img width="367" alt="screen shot 2017-08-05 at 17 54 51" src="https://user-images.githubusercontent.com/2543659/28996816-36d030e0-7a07-11e7-974d-f041b2b886a3.png">

Might be tricky to parse this, but I'm sure there are examples.